packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -852,6 +838,20 @@ const runGenerator = (
         // Install dependencies for the newly minted project.
         npmInstall(outputDir, {verbose, projectName: context.answers.project.name})
     }
+
+    // Convert selected plugins array to object with true values
+    if (Object.keys(pluginConfig?.plugins || {}).length > 0 && selectedPlugins) {
+        trimExtensions(outputDir, selectedPlugins)
+    }
+
+    // Compute the checksum of the output directory
+    const checksums = computeChecksum(outputDir)
+    const checksumFilePath = p.join(outputDir, 'checksum.json')
+    const timestamp = new Date().toISOString()
+    fs.writeFileSync(
+        checksumFilePath,
+        JSON.stringify({checksums, timestamp, selectedPlugins}, null, 2)
+    )
```

Comment:
Moving trim-extension to the very end so it doesn't trim files that haven't been finalized.